portal/celery_worker.py

```diff
@@ -10,17 +10,18 @@
   $ celery worker -A portal.celery_worker.celery --loglevel=info
 
 """
-from .app import create_app
 from .database import db
-from .extensions import celery
+from factories.celery import create_celery
+from factories.app import create_app
 from .models.scheduled_job import ScheduledJob
-import tasks
 
-assert celery  # silence pyflake warning
 
 app = create_app()
+celery = create_celery(app)
 app.app_context().push()
 
+# Todo: fix 'RuntimeError: Working outside of application context.'
```

Comment:
What is the source of this?  This will happen when you try and access something like `current_app` outside of a valid context - such as using it at file scope rather than within a function called during a valid context.